### PR TITLE
Note the requirement to grant access to orgs when authorizing the OAuth app

### PR DESCRIPTION
**Pull request by zimeg**
### Type of change

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

This PR adds a clarifying note that access to an organization might be required when using workflows with these organizations. Related to #45.

### Requirements

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +121,9 @@ the GitHub account you want to authenticate with:
 $ slack external-auth add
 ```
 
+To access repositories in an organization from a workflow, also grant access to
+the organization when authorizing your OAuth app.
```
Comment (misscoded) [inline README.md:125]:
```suggestion
**Note: when working with repositories that are part of an organization, be sure to grant access to
that organization when authorizing your OAuth app.**
```

**Review by misscoded — APPROVED** [review]
Suggestion for language. Thanks for seeing to this so promptly! 🙇🏼‍♀️

File: README.md (hunk 1 above)
Comment (zimeg) [reply README.md:125]: This reads much better, thank you!

**Comment by zimeg** [comment]
@misscoded and thank you for the prompt review! 🎉

**Pull request merged**